test-suite/src/test/java/io/micronaut/docs/http/server/cors/CorsController.java

```diff
@@ -0,0 +1,26 @@
+package io.micronaut.docs.http.server.cors;
+
+// tag::imports[]
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.http.annotation.Produces;
+import io.micronaut.http.server.cors.CrossOrigin;
+// end::imports[]
+
+// tag::controller[]
+@Controller("/hello")
+public class CorsController {
```

Comment:
annotate with `@Requires(property="spec.name"` to avoid bean pollution.

Reply:
resolved via resolved via https://github.com/micronaut-projects/micronaut-core/pull/8580/commits/717a22d209991591569b64857243d5c7b7a6ec41